nitro-protocol/test/contracts/LedgerFinancingApp/requireStateSupported.test.ts

```diff
@@ -129,17 +125,33 @@ describe('requireStateSupported', () => {
   it('accepts legitimate interest calculations', async () => {
     // construct a proof+candidate test case with fair interest calculation, assert passing    // with the interest rate applied.
     // test case:
-    // - appdata interest rate is 1% per day
-    // - initial outcome is 500:500, with 500 principal
-    // - 1 day passes
-    // - challenge outcome is 505:495
-    advanceOneDay();
+    // - appdata interest rate is 0.1% per block
+    // - initial outcome is 1000:0, with 1000 principal
+    // - 1 block passes
+    // - challenge outcome is 999:1
```

Comment:
Nit: these comments could become out of sync with the code in future. They could be removed.

Reply:
The eternal struggle.

I like these because the prose descriptions of the test are very short & comprehensible. The context "preload" makes the test code easier to read. In this case something *very close to* to spirit of these tests will remain the meatiest bit of this contract's test code, so I'm inclined to leave them for now.

Reply:
Fair enough!